lms/envs/common.py

```diff
@@ -2572,6 +2572,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
     # ADG Features app
     'openedx.adg.lms.applications',
+    'openedx.adg.common.custom_registration_form',
```

Comment:
- Let's please rename the app, as `form` in the name of app is confusing. A suggestion is `registration_extension` because we are actually extended edX's registration. Custom sounds like we have built something of our own.
- Let's please move this app in adg/lms because it would just be used in lms. We place those things in common which are being used for both in lms and cms.

Reply:
So Login isn't the same for lms and cms?